index.js

```diff
@@ -122,20 +123,20 @@ function handleArguments(env) {
 
   // Anything that needs to print outside of the logging mechanism should use console.log
   if (opts.version) {
-    console.log('CLI version', cliVersion);
-    if (env.modulePackage && typeof env.modulePackage.version !== 'undefined') {
-      console.log('Local version', env.modulePackage.version);
-    }
+    console.log('CLI version:', cliVersion);
+    console.log('Local version:', env.modulePackage.version || 'Unknown');
```

Comment:
I'm pretty sure there is the chance that `env.modulePackage` can be undefined, so we need to keep a check for that. It'll likely need an `/* istanbul ignore */` comment